readme.md

```diff
@@ -3,11 +3,68 @@
 [![Build Status](https://travis-ci.org/andela/suyabay.svg)](https://travis-ci.org/andela/suyabay)
 [![License](https://poser.pugx.org/andela/suyabay/license.svg)](LICENSE.md)
 
-Suyabay Podcast is an aggregator of podcasts for Suya lovers across the net. The purpose of this site is to help Suya lovers find new podcasts and contents. I encourage you to subscribe to the individual podcasts feeds and support the content creators.
+Suyabay Podcast is an open source project using the full features of `laravel`. The main purpose of this project is to help Suya lovers find new podcasts and contents.
+
+## Installation
+1. Clone the repository into your project folder
+2. Run composer install from `cmd ` to install all project dependencies
+3. Run ```php artisan migrate``` to install the database migration
+
+##Setup the environmental variables (.env file)
+
+        APP_ENV    =local
+        APP_DEBUG  =true
+        APP_KEY    =LhsswvmAfygWZdKUhZXedm3bOTAOKLxH
+
+        ### Database configuration
+        DB_HOST    =localhost
+        DB_DATABASE=
+        DB_USERNAME=
+        DB_PASSWORD=
+
+        ### Test database configuration
+        DB_TEST_HOST   =localhost
+        DB_TEST_DATABASE=
+        DB_TEST_USERNAME=
+        DB_TEST_PASSWORD=
```

Comment:
Give them values of `xxxxxxx` , then specify that they have to replace the value of `xxxxxx` with real values